shell/server/pre-meteor.js

```diff
@@ -123,23 +123,23 @@ Meteor.startup(function () {
         }
       });
     }).catch(function (err) {
-      res.writeHead(500, { "Content-Type": "text/plain" });
+      res.writeHead(500, { "Content-Type": "text/html" });
       res.end(err.message);
     });
   });
 });
 
 var errorTxtMapping = {};
-errorTxtMapping[Dns.NOTFOUND] = "\n" +
-    "If you were trying to connect this address to a Sandstorm app hosted at this server,\n" +
-    "you either have not set your DNS TXT records correctly or the DNS cache has not\n" +
-    "updated yet (may take a while).\n";
+errorTxtMapping[Dns.NOTFOUND] = "<br />\n" +
```

Comment:
`<br />` (and, in general, ending a tag with `/>`) is actually not valid HTML. It is valid XHTML, but contrary to popular belief XHTML is not a subset of valid HTML, and serving XHTML with `text/html` as the content type is incorrect. See: http://hixie.ch/advocacy/xhtml (<- this is the document that basically killed XHTML ~10 years ago. As it turns out, almost no one has ever actually used XHTML in production because almost everyone has always used `text/html` as the content type, but a lot of people _thought_ they were using XHTML.)

So, these should just be `<br>`.